Makefile

```diff
@@ -185,6 +185,9 @@ configure-testnet:
 	# Transfer ownership
 	$(TRANSFER_OWNERSHIP) $(CONFIGURE_ARGS_COMMON) --oapp-config $(CONFIG_BASE_PATH)/usdc-token.config.ts --signer deployer
 
+	# Transfer USDT ownership
+	$(TRANSFER_OWNERSHIP) $(CONFIGURE_ARGS_COMMON) --oapp-config $(CONFIG_BASE_PATH)/usdt-token.config.ts --signer deployer
```

Comment:
I am wondering if anything else needs to change in the makefile regarding USDT. For example,  in oft-token.config.ts I see [USDT-related configs for minters](https://github.com/stargate-protocol/stargate-v2/blob/46c7642a6e4aeff0cd1d6bfb4959dfd61918895b/packages/stg-evm-v2/devtools/config/mainnet/01/oft-token.config.ts#L168-L175) -- so should these now be ignored for external deployments or are the USDT configs in this file still needed?

Reply:
If we don't need those configs then we can remove them and the same goes for any other tokens - if they come preconfigured then we're good

Reply:
Do you mean remove them as in, not even add them to the configuration files when deploying stargate (i.e [the PRs](https://github.com/stargate-protocol/stargate-v2/pull/140/files) @stgLamps has been putting up)?

Reply:
Considering this will only be for chains moving forward now ^

For chains with existing deployments, like klaytn, we can remove the configs when we redeploy perhaps

Reply:
For any and all tokens that come preconfigured